esx_service/cli/vmdkops_admin.py

```diff
@@ -723,38 +731,341 @@ def get_version():
     except:
         return NOT_AVAILABLE
 
+def get_tenant_from_db(name):
+    auth.connect_auth_db()
+    error_info, tenant = auth._auth_mgr.get_tenant(name)
+    return error_info, tenant
+
+def operation_fail(error_info):
+    print error_info
+    return error_info
+
+def tenant_ls_headers():
+    """ Return column names for tenant ls command """
+    headers = ['Uuid', 'Name', 'Description', 'Default_datastore', 'VM_list']
+    return headers
+
+def generate_vm_list(vms):
+    """ Generate a comma separated string with given list """
+    res = ""
+    for vm in vms:
+        # vm[1] is vm_name, vm has format (vm_uuid, vm_name)
+        res = res + vm[1]
+        res = res + ","
+        
+    if res:
+        res = res[:-1]
+
+    return res    
+
+def generate_tenant_ls_rows(tenant_list):
+    """ Generate output for tenant ls command """
+    rows = []
+    for tenant in tenant_list:
+        uuid = tenant.id
+        name = tenant.name
+        description = tenant.description
+        default_datastore = tenant.default_datastore
+        vm_list = generate_vm_list(tenant.vms)
```

Comment:
New PR is filed to track this. https://github.com/vmware/docker-volume-vsphere/issues/651